httpmw/Ip_cognito_auth_test.go

```diff
@@ -65,11 +83,12 @@ func TestCognitoIP401(t *testing.T) {
 
 func TestCognitoIpAuth(t *testing.T) {
 	assert := assert.New(t)
-	ipRanges := "192.168.20.1-192.168.20.10"
-	token := authTestToken
+	ipRanges := authTestIPRanges
```

Comment:
This variable is redundant you can just pass `authTestIPRangesLarge` directly into  `IpCognitoAuth`.